added option to choose default/darker shimmer colors

ema987 opened the pull request:
fixes issue #20

ema987 committed:
added option to choose default/darker shimmer colors

elye reviewed (changes requested):
Good change. 
I suggest don't need to have 2 custom attributes (defaultColor, darkerColor), but instead of one (customColor). 
And when that exist, both bold and non bold would use that instead of the default one. 
If possible, make one of the text line in the sample app to use the customColor, and amend the readme to have the feature.
We want to minimize the custom attributes to make it easier to use.
Thanks.

ema987 committed:
removed default_color and darker_color attributes, now using only custom_color
If set, the custom_color attribute will override the default and darker colors specified in the library

ema987 committed:
using a custom_color in the example app

ema987 committed:
updated README to show how to use custom_color attribute

ema987 commented:
Hi @elye, thank you for your reply!
I've made changes following your suggestions. Code is basically as previous commits besides that now custom_color attribute will override both defaultColor and darkerColor used in the CustomViews.
I've updated the readme, too.
It should be ok now, but if it's not, just let me know!
